### ISSUE TITLE
Dead link for the blue pill support crate

### ISSUE THREAD

**Issue by gbip** [rust-embedded/awesome-embedded-rust#84]
Hello !
The blue pill support crate redirect to the [stm32f103 hal](https://github.com/japaric/stm32f103xx-hal).

**Comment by therealprof** [comment]
@gbip Why is that a problem and how would you like to see it addressed?

**Comment by gbip** [comment]
I think that this is misdirecting because one would expect the link to send us to a board support crate, while it is sending us to an hal implementation. Maybe we should remove it from this category ?

**Comment by therealprof** [comment]
I think @japaric idea here was that there's actually very little `blue-pill` specific in that crate that is not in the hal itself (because there's very little on the blue-pill board) so the hal can really take over the role of a board specific crate. I stumbled over the exact same redirect a while back but was not aware that this might be confusing.

@japaric Would it be okay for you if we removed the BSP crate from the list?
